public/tracking.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<link rel="icon" href="favicon.ico">
+
+		<title>OpenFisca - Turn law into software</title>
+
+		<link rel="stylesheet" type="text/css" href="css/main.css">
+		<link rel="stylesheet" type="text/css" href="css/tracking.css">
+	</head>
+	<body>
+		<img id="logo" src="logo-openfisca.svg" alt="OpenFisca">
+
+		<h1>Cookies and opt-out</h1>
+		<p>
+			When you visit this website, we leave a small text file (a "cookie") on your computer. This helps us measure how many visits we've had and which pages are most viewed.
+		</p>
+		<iframe class="optout" src="https://stats.data.gouv.fr/index.php?module=CoreAdminHome&action=optOut&language=en"></iframe>
+
+		<h2>This website does not show a consent banner for cookies, why?</h2>
+		<p>
+			Our tracking service, <a href="https://www.piwik.org">Piwik</a> is configured in compliance with the <a href="https://www.cnil.fr/fr/solutions-pour-la-mesure-daudience"  hreflang="fr-FR">"Cookies" recommandations (French)</a> from the French authorities (<abbr title="Commission Nationale de l'Informatique et des Libertés">CNIL</abbr>)
+			(i.e. Piwik anonymizes your IP address to make it impossible to link your visit to you personally).
+		</p>
+		<h2>I contribute to your data, can I access it?</h2>
+		<p>
+			Of course! The analytics data for OpenFisca is freely available on <a href="http://stats.data.gouv.fr/index.php?module=CoreHome&action=index&idSite=4&period=range&date=previous30#?module=Dashboard&action=embeddedIndex&idSite=4&period=range&date=previous30&idDashboard=1">stats.data.gouv.fr</a>.
+		</p>
+		<footer>
+			<ul>
+				<li><a href="tracking.html">Analytics policy</a></li>
```

Comment:
I don't need this link when I'm on the page.

Reply:
I think it is convenient that all footers have the same content. However I made it so the page we are on is not a link.

Reply:
I think the footer should be the same on every page, I find it disturbing that the link disappears when viewing the current page.